src/Search/Api/SearchController.php

```diff
@@ -134,6 +136,8 @@ public function indexAction(Request $request, Accept $type)
                 break;
         }
 
+        $query = $query->setClientOptions($this->clientOptions);
```

Comment:
Shouldn't be in the controller? Looking at the architecture, set in... `ElasticQueryExecutor`? (Which appears to be stateful. 😐) Though maybe `MappedElasticsearchClient` should actually take the query object and set it there instead.